ui/components/app/confirmation/confirm-footer/confirm-footer.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Please refer this PR for folder structure to be used: https://github.com/MetaMask/metamask-extension/pull/22081